packages/@okta/vuepress-site/docs/reference/api-overview/index.md

```diff
@@ -79,7 +79,34 @@ Okta will derive client request context directly from the HTTP request headers a
 
 Okta supports the standard `User-Agent` HTTP header to identify the user's browser or application. Always send a `User-Agent` string to uniquely identify your client application and version such as `Oktaprise/1.1`.
 
-> If your application is acting as a gateway or proxy, you should forward the `User-Agent` of the originating client with your API requests.
+> **Note:** If your application is acting as a gateway or proxy, you should forward the `User-Agent` of the originating client with your API requests.
+
+### Format a User-Agent string for native mobile apps
+
+Make sure that the User-Agent string that your app constructs is in the correct format. This ensures that Okta can parse the `OS` and `Browser` fields. Good User-Agent strings that can be correctly parsed by Okta contain browser information, system information, platform, platform details, and any extensions.
+
+#### Use a template to format the string
+
+We recommend that you use a template like the following to format the User-Agent string:
+
+`User-Agent: Mozilla/5.0 (<system-information>) <platform> (<platform-details>) <extensions>`
+
+#### Pass certain values in the string
+
+Alternatively, you can make sure that certain string values are present in the User-Agent string so that the OS and Browser are detected:
+
+> **Note:** Case doesn't matter since Okta converts the string to lowercase before matching.
+
+**Pass a hint about the browser**
+
+If you have a browser string such as `chrome` or `safari`, add that to the User-Agent string.
+
+**Pass operating system information**
+
+- **iOS:** Include the words `apple` or `ios` and at least one of these values: `iphone`, `ipad`, `ipod`, `ipad`.
+- **Android:** Include the words `android` or `samsung`, which infers that Android is the operating system.
+
+> **Note:** Okta recommends testing to make sure that Okta can parse both the `OS` and `Browser` fields from the User-Agent header that is passed by your native mobile app. See the [System Log](/docs/reference/api/system-log/#useragent-object).
```

Comment:
How would the developer know how to test? if we include this Note we should also describe or point to info on how to test. For ex, "to test if Okta can parse the string, check the System log of the request that was passed. See the [System Log] ... "